action.yml

```diff
@@ -29,14 +23,14 @@ inputs:
     description: "The version of Wrangler you'd like to use to publish your Workers project"
     required: false
   secrets:
-    description: "A new line deliminated string of environment variable names that should be configured as Worker secrets"
+    description: "Multiline or JSON - string of environment variable names that should be configured as Worker secrets"
     required: false
   preCommands:
     description: "Commands to execute before publishing the Workers project"
     required: false
   postCommands:
     description: "Commands to execute after publishing the Workers project"
     required: false
-  command:
-    description: "The Wrangler command you wish to run. For example: \"publish\" - this will publish your Worker"
-    required: false
\ No newline at end of file
+  commands:
```

Comment:
I think `commands` make sense here because I wouldn't have known otherwise (without looking into example actions or relying on a specific editor extension) that the field accepts more than one command.